test/tests/functional/pbs_resv_start_dur_end.py

```diff
@@ -90,3 +89,46 @@ def test_duration_end_resv_fail(self):
         self.assertTrue(rid is None,
                         'Reservation Submit succeeded' +
                         'when it should have failed')
+
+    def test_start_dur_end_resv_fail(self):
+        """
+        Test to submit a job with a start, end, and duration
+        where start + duration != end.
+        """
+        # create reservation to start in 10 seconds
+        # to end in 30 seconds
+        # and to last 10 seconds
+        now = int(time.time())
+        a = {'Resource_List.select': '1:ncpus=1',
+             'reserve_start': now + 10,
+             'reserve_end': now + 30,
+             'reserve_duration': 10}
+        R = Reservation(TEST_USER, attrs=a)
+        rid = None
+        try:
+            rid = self.server.submit(R)
+        except PbsSubmitError as e:
+            self.assertTrue('pbs_rsub: Bad time specification(s)' in e.msg[0],
```

Comment:
Don't include pbs_rsub's text in the message.  Only include the message.  This will fail in API mode.

Reply:
Ah, I didn't know that. Will fix.